### PR TITLE
Fix non-pushed changes

### PR DESCRIPTION
**Pull request by cochicde**
For some reason, the changes of the new fields weren't merged.

### CODE REVIEW

File: legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java [hunk 1]
```diff
@@ -26,18 +28,20 @@
 	public LegacyArrowheadService getProvidedService() { return providedService; }
 	public LegacyArrowheadSystem getProvider() { return provider; }
 	public String getServiceURI() { return serviceURI; }
-	public String getServiceUri() { return serviceURI; }
```
Comment (borditamas) [inline legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java:29]: Do not remove this. As described in the previous pull request both getter is necesarry for proper parsing as others may use it with lower case based on the 4.1.2 client skeleton.

File: legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java [hunk 2]
```diff
@@ -26,18 +28,20 @@
...
 	public Integer getVersion() { return version; }
+	public Integer getTtl() { return ttl; }
+	public String getMetadata() { return metadata; }
 	
 	//-------------------------------------------------------------------------------------------------
 	public void setId(final Long id) { this.id = id; }
 	public void setProvidedService(final LegacyArrowheadService providedService) { this.providedService = providedService; }
 	public void setProvider(final LegacyArrowheadSystem provider) { this.provider = provider; }
-	public void setServiceURI(final String serviceUri) { this.serviceURI = serviceUri; }
-	public void setServiceUri(final String serviceUri) { this.serviceURI = serviceUri; }
```
Comment (borditamas) [inline legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java:39]: Do not remove this. As described in the previous pull request both setter is necesarry for proper parsing as others may use it with lower case based on the 4.1.2 client skeleton.

**Review by borditamas — APPROVED** [review]
See inline comments. After fixing them your request will be merged.

File: legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java (hunk 2 above)
Comment (cochicde) [reply legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java:39]: If I leave the functions, the answers return with serviceURI and serviceUri:

`{
      "id": 12,
      "providedService": {
        "id": 7,
        "serviceDefinition": "service_test",
        "interfaces": [
          "HTTP-INSECURE-JSON"
        ],
        "serviceMetadata": {
          "key1": "value1",
          "key2": "value2",
          "security": "NOT_SECURE"
        }
      },
      "provider": {
        "id": 5,
        "systemName": "system_test_name",
        "address": "localhost",
        "port": 8090,
        "authenticationInfo": null
      },
      "serviceURI": "/serviceTest/start",
      "udp": false,
      "endOfValidity": null,
      "version": 1,
      "ttl": 0,
      "metadata": null,
      "serviceUri": "/serviceTest/start"
    },`

Comment (borditamas) [reply legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java:39]: Coudn't you just ignore the additinal field when reciving the response?

Comment (cochicde) [reply legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java:39]: I was thinking the same. I'll try to do it and come back here

Comment (borditamas) [reply legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java:39]: Is that worked?

Comment (cochicde) [reply legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java:39]: Yes, I forgot to answer. I added some code to omit the lowercase one

**Pull request merged**